dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/AddRemoveKpis.ts

```diff
@@ -36,46 +32,10 @@ describeIf(
         DataTypeEnum.EutaxonomyFinancials,
         testCompany,
         testData.t,
-        fillAndValidateEuTaxonomyForFinancialsUploadForm,
+        (data) => fillAndValidateEuTaxonomyCreditInstitutionForm(data),
         () => undefined,
         () => undefined
       );
     });
   }
 );
-
-/**
- * Fills the eutaxonomy-financials upload form with the given dataset
- * @param data the data to fill the form with
- */
-function fillAndValidateEuTaxonomyForFinancialsUploadForm(data: EuTaxonomyDataForFinancials): void {
```

Comment:
Same:
This can be reverted to main, because you don't need a form-filling-function anymore for your new test, if you just upload the stuff via the api there as we discussed today.

Reply:
This test needs the form to be filled out as it tests adding and removing of KPIs.. manually.

Reply:
Ok I just noticed you just moved it into the eu taxo financials utils file. That's ok for me 👍